src/modules/location/index.ts

```diff
@@ -38,9 +43,18 @@ export class LocationModule {
     options:
       | string
       | {
+          /**
+           * The state to generate the zip code for.
+           *
+           * Only works for locales with postcode_by_state definition. If a locale does not
+           * have a postcode_by_state definition an error is thrown.
```

Comment:
IMO this is redundant:

```suggestion
           * If the currrent locale does not have a corresponding `postcode_by_state` definition, an error is thrown.
```